### ISSUE TITLE
Samples not working on Android 7.0

### ISSUE THREAD

**Issue by Merichbier** [Azure/azure-spatial-anchors-samples#71]
Hi,

According to the only requirement that I found in the releases notes, Azure Spatial Anchors SDK only depends on ARCore 1.8.

I have a Zenphone on Android 7.0 which run ARCore > 1.8 and the samples fail to run because of 
` Unable to find azurespatialanchorsndk`
According to my logcats.

Note that the Unity project was set to Android 8.0 and I changed it to deploy on my Zenphone.

Either there is a bug and it should work, or the minimal requirement of Android 8.0 should be written somewhere...

**Comment by Merichbier** [comment]
I could get rid of this error by updating the dependencies in the package manager to :
    "com.unity.xr.arcore": "2.1.1",
    "com.unity.xr.arfoundation": "2.1.3"

But now it seems that the session failed to be created
This code seems to fail silently (when using some debug log at the end of this code it shows that the function work on Android 9 but not in Android 7
```C#
                AndroidHelper.Instance.DispatchUiThread(unityActivity =>
                {
                    // Create the plugin
                    using (AndroidJavaClass cloudServices = new AndroidJavaClass("com.microsoft.CloudServices"))
                    {
                        // Initialize the plugin
                        cloudServices.CallStatic("initialize", unityActivity);

                        // Update static variable to say that the plugin has been initialized
                        javaInitialized = true;

                        // Set the task completion source so the CreateSession method can
                        // continue back on the Unity thread.
                        pluginInit.SetResult(true);
                    }
                });
```

Any idea why it fails ?

**Comment by Merichbier** [comment]
Here is the message from the DispatchUiThread :

```java.lang.UnsatisfiedLinkError: JNI_ERR returned from JNI_OnLoad in "/data/app/<app-package-name>/lib/arm/libazurespatialanchorsndk.so"
```

**Comment by vriveras** [comment]
We have a fix for this issues coming in the next release.

**Comment by Merichbier** [comment]
Thanks for the update. Do you have any roadmap we can see about when this next release will be available ?

**Comment by craigktreasure** [comment]
Sorry about the delay. We just released version 2.0.0, which includes a fix for this. Please give 2.0.0 a try and feel free to re-open this if you have further issues.
